tuned/plugins/plugin_disk.py

```
@@ -199,43 +209,47 @@ def _instance_init_dynamic(self, instance):
+				log.info("disabling set_apm command: too many consecutive errors")
+		self._apm_errcnt = cnt
 
-	def _change_spindown(self, instance, device, new_spindown_level):
-		log.debug("changing spindown to %d" % new_spindown_level)
-		(rc, out) = self._cmd.execute(["hdparm", "-S%d" % new_spindown_level, "/dev/%s" % device], no_errors = [errno.ENOENT])
-		self._update_errcnt(rc, True)
-		instance._spindown_change_delayed[device] = False
+	def _try_set_spindown(self, device, level):
+		if not self._use_hdparm:
+			return False
+		if self._hdparm_apm_device_support.get(device) == False:
```

Ruff (0.16.2)

[error] Avoid equality comparisons to `False`; use `not self._hdparm_apm_device_support.get(device):` for false checks

Replace with `not self._hdparm_apm_device_support.get(device)`

(E712)